qml: Make links, data location and numerical description value un-tra…

shaavan opened the pull request:
This is a follow-up to #124 and addresses [this](https://github.com/bitcoin-core/gui-qml/pull/124#discussion_r930629844) comment.

This PR removed the `qsTr()` tag for the description strings, which were links, disk locations, or numerical values, and hence wouldn't be translated from their source language.

jarolrod commented on src/qml/components/AboutOptions.qml:
```diff
@@ -12,25 +12,25 @@ ColumnLayout {
         Layout.fillWidth: true
         header:  qsTr("Website")
-        description: qsTr("bitcoincore.org >")
+        description: "bitcoincore.org >"
```
while here, and working on a follow-up on this file, please seperate out the greater than symbol here. This shouldn't use a text symbol to represent the `right caret` icon as shown in the designs. the `right caret` icon is already in the repo.

jarolrod replied:
done in #144

shaavan replied:
Great! Thanks, Jarol!

shaavan commented:
Updated from b4fefbf1f8edf6372eb2df80144d2dc0f5ccdf48 to 7ff6ff2fd5f689527398295d8637a1817d4acab6 ([pr140.01](https://github.com/shaavan/gui-qml/commits/pr140.01) -> [pr140.02](https://github.com/shaavan/gui-qml/commits/pr140.02), [diff](https://github.com/shaavan/gui-qml/compare/pr140.01..pr140.02))

Changes:

- Rebased over main and resolved merge conflicts.

jarolrod commented on src/qml/components/ConnectionSettings.qml:
```diff
@@ -45,15 +45,15 @@ ColumnLayout {
         header: qsTr("Proxy settings")
         actionItem: ValueInput {
-            description: qsTr(">")
+            description: ">"
```
we're supposed to use the caret-right icon now ... #144

shaavan replied:
Addressed in 0b689ad874f9c4648d817657d948bb6ff03c79f5

jarolrod commented:
should rebase over changes on master

jarolrod commented:
should rebase and incorporate with changes done on master since last push

johnny9 commented:
concept ack

jarolrod commented:
Picked up in https://github.com/bitcoin-core/gui-qml/pull/217

hebasto commented:
> Picked up in #217

Closing this one.
